lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -1029,10 +1029,11 @@ namespace Js
                             // let desc = Reflect.getOwnPropertyDescriptor(obj, key);
                             Js::PropertyDescriptor desc;
                             BOOL ret = JavascriptOperators::GetOwnPropertyDescriptor(proxy, propertyName, scriptContext, &desc);
+                            const JsUtil::CharacterBuffer<WCHAR> propertyString = JsUtil::CharacterBuffer<WCHAR>(propertyName->GetString(), propertyName->GetLength());
```

Comment:
Nit: this can be shortened a bit, like so:

```c++
const JsUtil::CharacterBuffer<WCHAR> propertyString(propertyName->GetString(), propertyName->GetLength());
```

Reply:
Thanks for the feedback, learned about another C++ feature. I've incorporated your suggested change and amended my pull request.